src/searchForm/autoComplete/overlay/index.tsx

```diff
@@ -39,6 +39,13 @@ const StyledAutoComplete = styled(autoComplete)`
     border-radius: 0;
   }
 
+  & .kirk-textField-wrapper .kirk-loader {
+    position: absolute;
+    top: 12px;
+    bottom: 0;
```

Comment:
Why the bottom?

Reply:
Used it for a test, it is not needed anymore
Thanks :)